stylelint-config.json

```diff
@@ -1,5 +1,59 @@
 {
   "rules": {
+    "color-hex-case": "lower",
+    "color-no-invalid-hex": true,
+
+    "function-calc-no-unspaced-operator": true,
+    "function-comma-space-after": "always-single-line",
+    "function-comma-space-before": "never",
+    "function-name-case": "lower",
+    "function-url-quotes": "always",
+    "function-whitespace-after": "always",
+
+    "number-leading-zero": "always",
+    "number-no-trailing-zeros": true,
+    "number-zero-length-no-unit": true,
+
+    "string-no-newline": true,
+    "string-quotes": "double",
```

Comment:
tslint enforces single quotes... in ts. Personally I'd be quite opposed to switching html to single quotes. I have the feeling that double is much more prevalent in html, though I don't have anything to back that claim up with right now. For CSS I don't care which we use, but I think I naturally tend to write single-quotes.